### PR TITLE
fix: disappearing delete and radio check button 

### LINKED ISSUE #2894: Unexpected behaviour : deleting an Offline Project
**Describe the bug**

Unexpected behaviour after deleting an offline project. 
We cannot choose a project to open (Radio button disappear)

**To Reproduce**
1. Go to 'https://circuitverse.org/simulator'
2. Click on 'project>open Offline '
3. Delete a project
4. See an error

**Expected behavior**

It should remain the same ( radio button and the delete icon should not disappear ) 

**Screenshots**

https://user-images.githubusercontent.com/76155456/152520518-b15b7780-06cc-44d6-b9c4-0f90f89f7267.mov

**Device Information [optional]:**
 - OS: MacOs
 - Browser Chrome

**Additional context**
Also, update dialog box style for better UI

### PR DESCRIPTION
**Pull request by fillingtothemomo**
Fixes #2894

#### Describe the changes you have made in this PR -
In the file `public/js/data.js` there is a function `deleteOfflineProject` which has some missing components resulting in the disappearing of radio button and the delete icon changing to a cross icon after the delete project confirmation is made. So, I redefined the function to display delete icon as the cross icon was not consistent with the dialog box design and used the custom-radio class.

I also made some changes in `simulator/src/css/main.stylesheet.css` to improve the dialog box components UI by changing the border-radius to make it more uniform.

### Screenshots of the changes (If any) -

https://github.com/CircuitVerse/CircuitVerse/assets/122573982/191c1c6f-2b42-4566-8009-180c924920b7

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: simulator/src/css/main.stylesheet.css [hunk 1]
```diff
@@ -1349,7 +1349,7 @@ input:checked + .slider:before {
     background: transparent;
     color: white;
     line-height: inherit;
-    border-radius: 1px;
+    border-radius: 5px;
```
Comment (Prerna-0202) [inline simulator/src/css/main.stylesheet.css:1352]: This is not necessary. Kindly remove it.

File: simulator/src/css/main.stylesheet.css [hunk 2]
```diff
@@ -1482,7 +1482,7 @@ input:checked + .slider:before {
     margin: 4px;
     padding: 10px;
     background: transparent;
-    border-radius: 1px;
+    border-radius: 5px;
```
Comment (Prerna-0202) [inline simulator/src/css/main.stylesheet.css:1485]: This is also not necessary. Kindly remove it too.

**Comment by Prerna-0202** [comment]
Please resolve code climate issues too.

**Comment by fillingtothemomo** [comment]
hey @Prerna-0202 I believe I have made all the required changes and also resolved the code-climate issues.

**Review by Prerna-0202 — APPROVED** [review]

**Comment by fillingtothemomo** [comment]
@tanmoysrt I checked that the ruby rails cli run failed for the pr and I'm not able to find the cause of that. Can you help in telling what's the issue with the pr?

**Comment by tanmoysrt** [comment]
> @tanmoysrt I checked that the ruby rails cli run failed for the pr and I'm not able to find the cause of that. Can you help in telling what's the issue with the pr?

It's not any issue with your code

**Review by tanmoysrt — APPROVED** [review]
Working as expected !

**Pull request merged**